fido/exceptions.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+
+
+class BaseTimeoutError(Exception):
+    """
+    Base class for all errors due to timeouts.
+    """
+
+
+class ConnectTimeoutError(BaseTimeoutError):
+    """
+    Connection took too long to establish
+    """
+
+
+class HTTPTimeoutError(BaseTimeoutError):
+    """
```

Comment:
If the mapping is 1:1 between the twisted error and the fido error, why even have the fido error? It is no secret as to the dependency chain: Fido -> Crochet -> Twisted